### PR TITLE
Prepare beoi17

### PR DESCRIPTION
**Pull request by jganseman**
A few more website modifications to prepare for 2017. FAQ expanded too.

### CODE REVIEW

File: data/faq/nl.yml [hunk 1]
```diff
@@ -43,41 +46,80 @@
...
+        - Omdat computerklassen moeilijker te reserveren zijn, hopen we meer flexibiliteit in de praktische organisatie te kunnen toelaten.<br> 
+        Dit alles is afhankelijk van de uiteindelijke specificaties van het wedstrijdplatform, dat momenteel nog in ontwikkeling is. Tegen eind november zullen we concreter kunnen zijn.
     -
       question: Onder welke voorwaarden kan de eerste ronde in de school doorgaan ?
       answer: |
-        De eerste ronde kan in uw school worden georganiseerd indien aan al de volgende voorwaarden is voldaan:<br>
-        - Minstens 10 leerlingen van uw school moeten deelnemen, alle niveau's tesamen. <br>
-        - Er moet een lokaal beschikbaar zijn, geschikt voor een examen (voldoende plaats tussen de leerlingen, ...).<br>
+        De eerste ronde kan in uw school worden georganiseerd indien aan al de volgende voorwaarden is voldaan (noot: door de wijzingen in 2017 is het mogelijk dat dit later nog licht wordt aangepast):<br>
+        - Minstens 4 leerlingen van uw school moeten deelnemen, alle niveau's tesamen. <br>
```
Comment (smadbe) [inline data/faq/nl.yml]: Discussed at the GA: no minimum if no technical reason for it

File: data/faq/nl.yml [hunk 2]
```diff
@@ -43,41 +46,80 @@
...
         - Er moet constant toezicht zijn tijdens de volledige proef.<br>
         - Het reglement van de proef (geen GSM, ...) moet gerespecteerd worden. Het wordt u enkele dagen voor de proef toegezonden. <br>
-        - De proef moet doorgaan op hetzelfde moment als in de andere centra. Dat wil zeggen, de proef moet beginnen op woensdag 27 januari 2016 tussen 13u en 14u30, en moet 1u30 duren. <em>(uitzondering voor scholen die de biologie-olympiade organiseren: zie hieronder)</em><br>
-        - U moet zelf alle vragenlijsten afdrukken op dinsdagmiddag of woensdagvoormiddag, en ze veilig bewaren tot het begin van de proef. <br>
-        - U moet ons de ingevulde opgaves (en zeker alle antwoordbladen) nadien doorsturen, ofwel door ze in te scannen en te emailen, ofwel via aangetekende post, ofwel door ze zelf af te geven in het verbetercentrum in Brussel (ULB Pleinlaan).
+        - De proef moet doorgaan op hetzelfde moment als in de andere centra. Voor editie 2017 betekent dit dat de proef plaatsvindt op woensdag 1 februari in de namiddag. De proef moet 1u30 duren. Er wordt later nog beslist hoeveel marge we daarop zullen toelaten.<br>
```
Comment (smadbe) [inline data/faq/nl.yml]: discussed at the GA: the afternoon is probably too limiting (if they cannot put all students in the room which is probably the case). Letting them participate all the day make more sense.

File: data/faq/nl.yml [hunk 3]
```diff
@@ -43,41 +46,80 @@
...
+        - Als een school de eerste ronde in eigen huis organiseert, moeten alle deelnemers van die school de eerste ronde op school afleggen. Een deel van de leerlingen op school en een ander deel in een regionaal centrum is niet mogelijk.<br>
```
Comment (smadbe) [inline data/faq/nl.yml]: Why ? It is not a problem after all, it is transparent, while checking this rule will be boring. Typically, if the teacher of 6th grade organise it for its full class during the CS course, and 2 guys in the 1st grade wants to participate, I've no problem they go to a center.

File: data/faq/nl.yml [hunk 4]
```diff
@@ -43,41 +46,80 @@
...
+        - Alle deelnemers aan de Informatica-olympiade van uw school die behoren tot eenzelfde categorie, beginnen op hetzelfde moment aan hun proef (niet enkelen om 14u en anderen om 15u).<br>
```
Comment (smadbe) [inline data/faq/nl.yml]: Again, just impossible to organise. Anyway, the questions will be the same (or almost) among levels. Let's trust the teachers, they organise exams already !

File: data/faq/nl.yml [hunk 5]
```diff
@@ -43,41 +46,80 @@
...
-        - De deelnemers hebben vanaf 14u30 geen contact met de buitenwereld: scholieren geven gsm, smartwatches en andere apparaten met internetverbinding af, met uitzondering van wat nodig is om de biologie-olympiade te beëindigen.<br>
-        - Alle leerlingen die deelnemen aan de Informatica-olympiade beginnen op hetzelfde moment (niet enkelen om 14u en anderen om 15u).<br>
-        
-        Wij vinden deze overlap met een andere wedstrijd zelf jammer, maar ze was niet te vermijden. Ze is een gevolg van het feit dat we als nationale olympiade rekening moeten houden met de verschillende kalenders van andere wedstrijden in zowel Vlaanderen als in Franstalig België. 
+        In 2017 valt de eerste ronde van de Informatica-olympiade samen met die van de Biologie-olympiade. De biologie-olympiade gaat ook door op school in een computerklas, op vaste tijdstippen. Uw leerlingen mogen voordien of na afloop de Informatica-olympiade afleggen.<br>
```
Comment (smadbe) [inline data/faq/nl.yml]: not relevant anymore if any moment during the day is ok

File: data/faq/nl.yml [hunk 6]
```diff
@@ -43,41 +46,80 @@
...
+        Wij vinden deze overlap met een andere wedstrijd zelf jammer, maar ze is niet te vermijden. Ze is een gevolg van de langere voorbereidingstijd die we nodig hebben in 2017, en het feit dat we als nationale olympiade rekening moeten houden met de kalenders van andere wedstrijden in zowel Vlaanderen als in Franstalig België. 
+    -
+      question: Is een computerklas noodzakelijk?
+      answer: De eerste ronde is in 2017 gepland als een wedstrijd via de computer. Indien de voorbereidingen daarvoor succesvol zijn, zal de wedstrijd in een computerklas met internetverbinding moeten plaatsvinden.
     -
-      question: Moet er aan de computer gewerkt worden in de eerste ronde ?
-      answer: "Nee, de eerste ronde is enkel een papieren examen."
+      question: Mijn computerklas is niet groot genoeg!
+      answer: |
+        Het zal zeer waarschijnlijk toegelaten worden de eerste ronde in maximaal 3 aparte sessies te organiseren, m.n. 1 sessie per deelnemerscategorie.
```
Comment (smadbe) [inline data/faq/nl.yml]: Again, it means that you reject Heilig Graf Turnhout school who asked to have 230 contestants (all in the same category) ?

File: data/faq/nl.yml [hunk 7]
```diff
@@ -43,41 +46,80 @@
...
+        We zullen echter wel vragen dat alle deelnemers van eenzelfde categorie en eenzelfde school, de wedstrijd tegelijkertijd afleggen.
+        Ook vragen we in dat geval dat de communicatie tussen deelnemers van verschillende sessies tot een absoluut minimum wordt beperkt, zelfs al behoren ze tot een verschillende categorie.
     -
       question: Moeten mijn leerlingen zich op uw site inschrijven? 
       answer: |
-        Alle deelnemers moeten zich persoonlijk inschrijven op onze site, zodat ze via email hun resultaten toegestuurd krijgen, en als ze in aanmerking komen, ook de uitnodigingen voor de vormingssessies en de finale krijgen.
+        We trachten vanaf 2017 de inschrijvingen te kunnen laten verlopen via het wedstrijdplatform. De details van de inschrijvingsprocedure worden momenteel nog uitgewerkt. 
+        Een geldig emailadres blijft echter wel noodzakelijk, zodat we deelnemers hun resultaten kunnen toesturen, en als ze in aanmerking komen, ook de uitnodigingen voor de vormingssessies en de finale kunnen bezorgen.
```
Comment (smadbe) [inline data/faq/nl.yml]: I think the French bebras platform does not include that... actually I think they do not even register the student names before, the students just has to provide their name when participating (the teacher can still check and fix typos during/after the contest)... let's use France experience which is much bigger than our. So let's not put this rule for now.

File: data/faq/nl.yml [hunk 8]
```diff
@@ -43,41 +46,80 @@
...
         Als per ongeluk een leerling niet ingeschreven blijkt te zijn op de dag van de eerste ronde, laten we wel toe om achteraf nog in te schrijven. 
     -
       question: Hoe schrijf ik mijn school in? 
       answer: |
-        Als u de intentie heeft de eerste ronde in uw school te organiseren, laat ons dat dan weten door een e-mail te sturen naar info@be-oi.be . Vraag ook aan uw leerlingen om zich in te schrijven voor de wedstrijd via deze website, voor 25 januari 2016. 10 leerlingen van uw school moeten ingeschreven zijn voordat we de organisatie van de eerste ronde in uw school bevestigen.
+        De details van de inschrijvingsprocedure voor scholen hopen we eind november te kunnen publiceren.
```
Comment (smadbe) [inline data/faq/nl.yml]: I've also added an additional question: do they need fast internet -> no

Comment (smadbe) [inline data/faq/nl.yml]: and about what is needed on the computer

File: data/faq/nl.yml (hunk 1 above)
Comment (jganseman) [inline data/faq/nl.yml]: OK!

File: data/faq/nl.yml (hunk 3 above)
Comment (jganseman) [inline data/faq/nl.yml]: Well, yes, that works if they are from different grades. But if from the same class, half of the students do it in school in the morning, and the other half of the class goes to a center in the afternoon, there's an obvious problem. So, rephrasing: all students of the same school and same grade must do it at the same location.

File: data/faq/nl.yml (hunk 8 above)
Comment (jganseman) [inline data/faq/nl.yml]: Updated

File: data/faq/nl.yml (hunk 7 above)
Comment (jganseman) [inline data/faq/nl.yml]: OK, adapted.

File: data/faq/nl.yml (hunk 6 above)
Comment (jganseman) [inline data/faq/nl.yml]: Let's not, after years of carefully monitoring potential cheating, suddenly go all wild wild west and allow just anything. Some consistency would be nice. I've rephrased it to make separate sessions possible, and added a line about exceptions to the rule if there are some guarantees, but I think we should still recommend a default setting: that everyone from the same school and same grade, starts at the same time in the same location.

File: data/faq/nl.yml (hunk 2 above)
Comment (jganseman) [inline data/faq/nl.yml]: OK!

File: data/faq/nl.yml (hunk 5 above)
Comment (jganseman) [inline data/faq/nl.yml]: might be relevant for flanders, adapted answer to ensure no problem.

**Comment by jganseman** [comment]
I've adapted it to most of the comments, replies are in the individual comments. Agreed with virtually everything, and have rephrased the sessions thing to something along the lines of: by default, same category does same location and same time, but exceptions are allowed.


-- J.

________________________________
Van: Damien Leroy <notifications@github.com>
Verzonden: woensdag 21 september 2016 22:58:21
Aan: be-oi/beoi.be-oi.be
CC: Ganseman Joachim; Author
Onderwerp: Re: [be-oi/beoi.be-oi.be] Prepare beoi17 (#36)


@smadbe requested changes on this pull request.

As indicated in the comments, some points have to be agreed before pushing anything. For the moment nl and fr version are not consistent.

-

**Pull request merged**